source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,13 @@
+// start-specify-v1
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1')];
+$client = new MongoDB\Client($uri, [], $driverOptions);
+// end-specify-v1
+
+// start-stable-api-options
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1', strict: true, deprecationErrors: true)];
```

Comment:
If I wanted to split this line into multiple, shorter lines, how would I do that?

Reply:
I think this would be readable:

```php
$uri = "mongodb://<hostname>:<port>";

$serverApi = new MongoDB\Driver\ServerApi('1', strict: true, deprecationErrors: true);
$driverOptions = ['serverApi' => $serverApi];

$client = new MongoDB\Client($uri, [], $driverOptions);
```

I also think the previous example could benefit from an extra newline before `$client = ...`.

----

If you want to break ServerApi across multiple lines, you can do something like:

```php
$serverApi = new MongoDB\Driver\ServerApi(
    '1',
    strict: true,
    deprecationErrors: true,
);
```

PHP 8.0 allows trailing commas in parameter lists (consistent with what we'd do for multi-line arrays): https://php.watch/versions/8.0/trailing-comma-parameter-use-list

If that's too much wrapping, you could also do this:

```php
$driverOptions = [
    'serverApi' => new MongoDB\Driver\ServerApi('1', strict: true, deprecationErrors: true),
];
```

That said, I don't think either of these are better than just assigning `$serverApi` on its own line as I did in the first suggestion.